src/plasmapy/particles/particle_class.py

```diff
@@ -783,6 +785,12 @@ def _assign_atom_attributes(self) -> None:
 
         categories.add(this_element["category"])
 
+        try:
+            ev_value = this_element["ionization energy"] * u.eV
+            attributes["ionization energy"] = u.Quantity(ev_value.to(u.J).value, u.J)
```

Comment:
Same comment as below, I think this is simpler
```suggestion
            attributes["ionization energy"] = ev_value.to(u.J)
```